pubnub/event_engine/effects.py

```diff
@@ -58,11 +58,11 @@ def get_new_stop_event(self):
         return event
 
     def calculate_reconnection_delay(self, attempts):
-        if self.reconnection_policy is PNReconnectionPolicy.LINEAR:
+        if self.reconnection_policy is PNReconnectionPolicy.EXPONENTIAL:
+            delay = int(math.pow(2, attempts - 5 * math.floor((attempts - 1) / 5)) - 1)
```

Comment:
Is the idea to use legacy exponential policy here? Do you plan to introduce an aligned exponential policy later?

Reply:
yes.

Reply:
I mean aligned policy will be implemented in the future